test/specs/apexReplayDebugger.e2e.ts

```diff
@@ -0,0 +1,327 @@
+      'Execute Anonymous Apex successfully ran',
+      utilities.Duration.TEN_MINUTES
+    );
+    expect(successNotificationWasFound).to.equal(true);
+
+    // Verify content on vscode's Output section
+    const outputPanelText = await utilities.attemptToFindOutputPanelText('Apex', 'Starting Execute Anonymous Apex', 10);
+    expect(outputPanelText).to.not.be.undefined;
+    expect(outputPanelText).to.contain('Compiled successfully.');
+    expect(outputPanelText).to.contain('Executed successfully.');
+    expect(outputPanelText).to.contain('|EXECUTION_STARTED');
+    expect(outputPanelText).to.contain('|EXECUTION_FINISHED');
+    expect(outputPanelText).to.contain('ended Execute Anonymous Apex');
+  });
+
+  step('SFDX: Get Apex Debug Logs', async () => {
+    utilities.log(`ApexReplayDebugger - SFDX: Get Apex Debug Logs`);
+
+    // Run SFDX: Get Apex Debug Logs
+    const workbench = utilities.getWorkbench();
+    await utilities.clearOutputView();
+    await utilities.pause(utilities.Duration.seconds(2));
+    prompt = await utilities.executeQuickPick('SFDX: Get Apex Debug Logs', utilities.Duration.seconds(0));
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway('Getting Apex debug logs', utilities.Duration.TEN_MINUTES);
+    await utilities.pause(utilities.Duration.seconds(2));
+
+    // Select a log file
+    const quickPicks = await prompt.getQuickPicks();
+    expect(quickPicks).to.not.be.undefined;
+    expect(quickPicks.length).to.be.greaterThanOrEqual(0);
+    await prompt.selectQuickPick('User User - Api');
+
+    const successNotificationWasFound = await utilities.notificationIsPresentWithTimeout(
+      'SFDX: Get Apex Debug Logs successfully ran',
+      utilities.Duration.TEN_MINUTES
+    );
+    expect(successNotificationWasFound).to.equal(true);
+
+    // Verify content on vscode's Output section
+    const outputPanelText = await utilities.attemptToFindOutputPanelText(
+      'Apex',
+      'Starting SFDX: Get Apex Debug Logs',
+      10
+    );
+    expect(outputPanelText).to.not.be.undefined;
+    expect(outputPanelText).to.contain('|EXECUTION_STARTED');
+    expect(outputPanelText).to.contain('|EXECUTION_FINISHED');
+    expect(outputPanelText).to.contain('ended SFDX: Get Apex Debug Logs');
+
+    // Verify content on log file
+    const editorView = workbench.getEditorView();
+    const activeTab = await editorView.getActiveTab();
+    const title = await activeTab?.getTitle();
+    const textEditor = (await editorView.openEditor(title!)) as TextEditor;
+    const executionStarted = await textEditor.getLineOfText('|EXECUTION_STARTED');
+    const executionFinished = await textEditor.getLineOfText('|EXECUTION_FINISHED');
+    expect(executionStarted).to.be.greaterThanOrEqual(1);
+    expect(executionFinished).to.be.greaterThanOrEqual(1);
+  });
+
+  xstep('SFDX: Launch Apex Replay Debugger with Last Log File', async () => {
```

Comment:
I see this is still as xstep, I thought we said in standup that we would make it run as we won't continue trying to make ubuntu/windows pass

Reply:
That step doesn't work on Mac - the filepath gets truncated when pasted in the prompt box.  My [first passing run](https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/11768719011/job/32778851658) was with the xstep.

Reply:
Later today I'll take a look at this then, we don't want to lose this use case too

Reply:
Use case passing in https://github.com/forcedotcom/salesforcedx-vscode/actions/runs/11823142221/job/32941797011 🚀